examples/pipelines/simple_pipeline/components/load_from_hub/src/main.py

```diff
@@ -11,7 +11,7 @@
 import numpy as np
 from PIL import Image
 
-from fondant.dataset import FondantComponent
+from fondant.dataset import FondantLoaderComponent
```

Comment:
```suggestion
from fondant.dataset import FondantLoadComponent
```
Let's call them "load" and "transform"

Reply:
done :)